middlewares/studentAssessmentAccess.js

```diff
@@ -32,12 +32,45 @@ router.get('/', function(req, res, next) {
         }
     }
 
+    // Password protect the assessment
```

Comment:
This is in a `router.get()` handler. Should it be `router.any()` so we also run these checks on POST requests?